server/changes/VERSION_COMPAT

```diff
@@ -0,0 +1 @@
+platform>=0.6.1
```

Comment:
only python packages make sense here, I was referring to the compat section in readme